webpack.common.js

```diff
@@ -41,7 +41,12 @@ module.exports = {
           },
           {
             loader: 'sass-loader',
-            options: { sourceMap: !PRODUCTION }
+            options: {
+              sourceMap: !PRODUCTION,
+              sassOptions: {
+                outputStyle: 'expanded'
```

Comment:
We should probably use `compressed` for prod?
```suggestion
                outputStyle: PRODUCTION ? 'compressed' : 'expanded'
```

Reply:
Fixed it!